src/components/BackButton/StyledBackButton.tsx

```diff
@@ -0,0 +1,20 @@
+import { Button } from '..';
+import styled from 'styled-components';
+import BackLight from '../../views/HowToPlay/assets/BackLight.svg';
+import colors from '@ui/styles/colors';
+
+export const StyledBackButton = styled(Button)`
+  border-radius: 5px;
+  color: ${colors.tan};
+  font-size: 20px;
+  background-image: url(${BackLight});
+  background-size: 1.25em;
+  background-repeat: no-repeat;
+  background-position-y: center;
+  background-position-x: 0.5em;
+  padding-left: 2.5em;
+  padding-right: 1em;
+  margin-top: 5vh;
+  margin-left: 3vw;
```

Comment:
Don't put a margin on this component. We can position it properly using a Layout Route.

```suggestion
```